### DISCUSSION TITLE
backpack_url returns http and not https

### DISCUSSION THREAD

**Discussion by lieto-dk** [Laravel-Backpack/community-forum#447] [Q&A (Help)] [answered]
Hi i have a strange problem.

I am setting up a new production environment and in that environment backpack_url returns http instead of https, so for instance the login form. So for instance the post form is set to

<form class="col-md-12 p-t-10" role="form" method="POST" action="http://test.lastminutecharter.eu/admin/login">

so the backpack_url helper get http and not https

My ENV file hast these variables set.

APP_URL=https://test.lastminutecharter.eu
APP_URI=https://test.lastminutecharter.eu
ASSET_URL=https://test.lastminutecharter.eu

Anyone have a solution or direction where i should find a solution to this problem ?

**Comment by lieto-dk** [comment]
Ups .. seems like the code from the form didnt make it so here goes
` <form class="col-md-12 p-t-10" role="form" method="POST" action="http://test.lastminutecharter.eu/admin/login">`
Assets work fine.
The web server is behind a load balancer.

**Comment by lieto-dk** [comment, marked as answer]
Hi, i found the answer, my Trusted proxies wasnt set correctly. Now i just need a best practice for handling trusted proxies across dev, staging and production .. but this is probably not the forum for this !

**Reply by tabacitu** [reply]
Glad you found your answer @lieto-dk . Thanks for sharing here - might help others who encounter the same problem.
